pkg/settings/setting.go

```diff
@@ -49,6 +49,7 @@ var (
 	SystemMonitoringCatalogID       = NewSetting("system-monitoring-catalog-id", "catalog://?catalog=system-library&template=rancher-monitoring&version=0.0.1")
 	AuthUserInfoResyncCron          = NewSetting("auth-user-info-resync-cron", "0 0 * * *")
 	AuthUserInfoMaxAgeSeconds       = NewSetting("auth-user-info-max-age-seconds", "3600") // 1 hour
+	APIUIVersion                    = NewSetting("api-ui-version", "1.1.6")
```

Comment:
Looks like will we need to keep this in sync in two places: here and in package/Dockerfile.
Can you add a comment to both places so that in the future when someone updates this, they know they need to update it in the other place

Reply:
Updated